src/explore-education-statistics-admin/src/pages/methodology/edit-methodology/content/MethodologyContentPage.tsx

```diff
@@ -38,33 +46,122 @@ const MethodologyContentPage = ({
             },
           ]}
           onChange={event => {
-            setPageMode(event.target.value as PageMode);
+            setIsEditing(event.target.value === 'edit');
           }}
         />
       </FormFieldset>
       <EditingContext.Provider
         value={{
-          isEditing: pageMode === 'edit',
+          isEditing,
           isCommenting: false,
           isReviewing: false,
           releaseId: methodology.id,
           availableDataBlocks: [],
         }}
       >
-        <main
-          className="govuk-main-wrapper app-main-class"
-          id="main-content"
-          role="main"
+        <div
+          className={classNames('govuk-width-container', {
+            'dfe-align--comments': isEditing,
+            'dfe-hide-comments': !isEditing,
+          })}
         >
-          <h1
-            className="govuk-heading-xl"
-            data-testid={`page-title ${methodology.title}`}
-          >
-            {methodology.title}
-          </h1>
-          <p>Lorem ipsum.</p>
-          <p>{JSON.stringify(methodology)}</p>
-        </main>
+          <section className={isEditing ? 'page-editing' : 'page-preview'}>
```

Comment:
Shouldn't these classes be namespaced i.e. `dfe-page-editing` and `dfe-page-preview`?